pkg/registry/serving/composer.go

```diff
@@ -273,14 +279,21 @@ func getModelFormat(pu *seldonv1.PredictiveUnit) string {
 
 // getUserContainerImage get image by different model format.
 func getUserContainerImage(format string) string {
-	if format == string(modeljobsv1alpha1.FormatPMML) {
+	switch format {
+	// Group1 for PMML image
+	case string(modeljobsv1alpha1.FormatPMML):
 		return viper.GetString(envPMMLServingImage)
-	}
-	if format == string(modeljobsv1alpha1.FormatSKLearn) || format == string(modeljobsv1alpha1.FormatXGBoost) {
+
+	// Group2 for mlserver image
+	case string(modeljobsv1alpha1.FormatSKLearn):
+		fallthrough
+	case string(modeljobsv1alpha1.FormatXGBoost):
 		return viper.GetString(envMLServerImage)
-	}
 
-	return viper.GetString(envTRTServingImage)
+	// Group3 for TRT server image
+	default:
+		return viper.GetString(envTRTServingImage)
+	}
```

Comment:
For MLflow, we need not set image, it will be set by seldon core? Is that right?

Reply:
yes, it will be set by Seldon core